modules/nf-core/bcftools/consensus/main.nf

```diff
@@ -20,6 +20,7 @@ process BCFTOOLS_CONSENSUS {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
+    def masking = mask ? "-m $mask" : ""
```

Comment:
You are not using this in the command below.

Reply:
you are correct, thanks for pointing out. I messed up my commits with this one :I. I'll correct that!